lib/matplotlib/backends/backend_gtk3.py

```diff
@@ -821,6 +821,15 @@ def _normalize_shortcut(self, key):
 
         return ''.join(mods) + key
 
+    def _is_valid_shortcut(self, key):
```

Comment:
Can you pleas add a docstring what "valid" means in this context and explain why keys can be invalid shortcuts?

Reply:
Added.